src/Form/ChoiceList/ModelChoiceLoader.php

```diff
@@ -56,36 +59,23 @@ final class ModelChoiceLoader implements ChoiceLoaderInterface
      */
     private $choices;
 
-    /**
-     * @var PropertyPath|null
-     */
-    private $propertyPath;
-
-    /**
-     * @var PropertyAccessorInterface
-     */
-    private $propertyAccessor;
-
     /**
      * @var ChoiceListInterface|null
      */
     private $choiceList;
 
     /**
-     * @param string      $class
-     * @param string|null $property
-     * @param object|null $query
-     * @param object[]    $choices
+     * @param object[] $choices
      *
-     * @phpstan-param class-string $class
+     * @phpstan-param class-string|null $class
      */
     public function __construct(
```

Comment:
Upgrade note is needed

Reply:
Do you think this case is different from the rest of the code we changed? TBH using this modelType inside sonata can't lead to getting null here as class

Reply:
The construct arguments changed.

Before `new ModelChoiceLoader($modelManager, $class)` was ok.
Now it won't work.